.travis.yml

```diff
@@ -20,7 +26,7 @@ r_github_packages:
   - USGS-R/rloadest@v0.4.2
 
 after_success:
-  - Rscript -e 'covr::codecov(); covr::coveralls()'
+  - Rscript -e 'library(covr);covr::codecov(); covr::coveralls()'
```

Comment:
How should we do that, should I do another commit, and make a new pull request? I'll wait to commit until you had a change to review everything else, just let me know if we are good to go.